src/main/java/com/redhat/devtools/intellij/lsp4ij/operations/documentation/LSPDocumentationProvider.java

```diff
@@ -0,0 +1,208 @@
+        }
+        return null;
+    }
+
+    private static int getTargetOffset(PsiElement originalElement) {
+        Integer targetOffset = originalElement.getUserData(TARGET_OFFSET_KEY);
+        if (targetOffset != null) {
+            return targetOffset;
+        }
+        int startOffset = originalElement.getTextOffset();
+        int textLength = originalElement.getTextLength();
+        return startOffset + textLength / 2;
+    }
+
+    @Nullable
+    @Override
+    public PsiElement getDocumentationElementForLookupItem(PsiManager psiManager, Object object, PsiElement element) {
+        if (object instanceof LSPCompletionProposal) {
+            MarkupContent documentation = ((LSPCompletionProposal) object).getDocumentation();
+            if (documentation != null) {
+                return new LSPPsiElementForLookupItem(documentation, psiManager, element);
+            }
+        }
+        return null;
+    }
+
+    @Nullable
+    @Override
+    public PsiElement getDocumentationElementForLink(PsiManager psiManager, String link, PsiElement context) {
+        return null;
+    }
+
+    @Override
+    public boolean handleExternal(PsiElement element, PsiElement originalElement) {
+        return false;
+    }
+
+    @Override
+    public boolean handleExternalLink(PsiManager psiManager, String link, PsiElement context) {
+        VirtualFile file = LSPIJUtils.findResourceFor(link);
+        if (file != null) {
+            FileEditorManager.getInstance(psiManager.getProject()).openFile(file, true, true);
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean canFetchDocumentationLink(String link) {
+        return false;
+    }
+
+    @Override
+    public @NotNull String fetchExternalDocumentation(@NotNull String link, @Nullable PsiElement element) {
+        return null;
+    }
+
+
+    public static String styleHtml(Editor editor, String html) {
+        if (html == null || html.isEmpty()) {
+            return html;
+        }
+        Color background = editor.getColorsScheme().getDefaultBackground();
+        Color foreground = editor.getColorsScheme().getDefaultForeground();
+        // put CSS styling to match Eclipse style
+        String style = "<html><head><style TYPE='text/css'>html { " + //$NON-NLS-1$
+                (background != null ? "background-color: " + toHTMLrgb(background) + "; " : "") + //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+                (foreground != null ? "color: " + toHTMLrgb(foreground) + "; " : "") + //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+                " }</style></head><body>"; //$NON-NLS-1$
+
+        /*int headIndex = html.indexOf(HEAD);
+        StringBuilder builder = new StringBuilder(html.length() + style.length());
+        builder.append(html.substring(0, headIndex + HEAD.length()));
+        builder.append(style);
+        builder.append(html.substring(headIndex + HEAD.length()));
+        return builder.toString();*/
```

Comment:
do we need to keep this commented code?

Reply:
I removed it and clean the code of this existing method.